mypy.ini

```diff
@@ -478,7 +478,7 @@ ignore_errors = True
 ignore_errors = True
 
 [mypy-scipy.spatial._spherical_voronoi]
```

Comment:
You can delete this part of the ratchet now.

Reply:
done